src/server/string_family.cc

```diff
@@ -620,6 +622,17 @@ void SetCmd::RecordJournal(const SetParams& params, string_view key, string_view
   dfly::RecordJournal(op_args_, "SET", ArgSlice{cmds});
 }
 
+Responder* StringFamily::TestResponder1(CmdArgList args, ConnectionContext* cntx) {
+  SimpleResponder<long>* rsp = cntx->MakeResponder<SimpleResponder<long>>();
```

Comment:
I decided to use this more implicit approach for responder creation, but we can also inject it.

For normal cases, we can store it inside some inline buffer (which I do currently), if we run on a fake-context for paralellizing eval/pipelines, we can hand out pooled buffers

Reply:
I am guessing this function needs to create a responder for any scenario. Are you planning to add conditions here for multiple buffers and a single one?